config/articles.json

```diff
@@ -0,0 +1,56 @@
+[
```

Comment:
This file targets the extra articles and recent posts that are to be added in the latest news section. Right now, it is in test mode, hence repeated data is being used to test the components.